stubs/colorful/colorful/terminal.pyi

```diff
@@ -0,0 +1,7 @@
+NO_COLORS: int
+ANSI_8_COLORS: int
+ANSI_16_COLORS: int
+ANSI_256_COLORS: int
+TRUE_COLORS: int
+
+def detect_color_support(env: dict[str, str]) -> int: ...
```

Comment:
Interestingly, we don't have a `SupportsGet` in `_typeshed`, but we can build on ourselves:

```suggestion
@type_check_only
class _SupportsGet(Protocol):
    @overload
    def get(self, name: str, /) -> str | None: ...
    @overload
    def get(self, name: str, default: str, /) -> str: ...

def detect_color_support(env: _SupportsGet) -> int: ...
```